homeassistant/components/yale_smart_alarm/strings.json

```diff
@@ -1,28 +1,43 @@
 {
   "config": {
     "abort": {
-      "already_configured": "[%key:common::config_flow::abort::already_configured_account%]"
+      "already_configured": "[%key:common::config_flow::abort::already_configured_account%]",
+      "reauth_successful": "[%key:common::config_flow::abort::reauth_successful%]"
     },
     "error": {
-      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]"
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]"
     },
     "step": {
       "user": {
         "data": {
           "username": "[%key:common::config_flow::data::username%]",
           "password": "[%key:common::config_flow::data::password%]",
-          "name": "[%key:common::config_flow::data::name%]",
-          "area_id": "Area ID"
+          "name": "[%key:component::yale_smart_alarm::config::step::user::data::name%]",
```

Comment:
? This now points to where? itself?

Reply:
Very strange as I have not changed that one. Submit changes now so hope this one gets resolved then